roles/openshift_master/tasks/main.yml

```diff
@@ -25,7 +25,13 @@
 
 - name: Set master facts
   openshift_facts:
-    role: master
+    role: "{{ item.role }}"
+    local_facts: "{{ item.local_facts }}"
+  with_items:
+  - role: common
+    local_facts:
+      portal_net: "{{ openshift_master_portal_net | default(None) }}"
```

Comment:
This should be pushed down into openshift_common, we should start avoiding setting role facts outside of the role itself.